src/lib.rs

```diff
@@ -555,6 +561,16 @@ where
         self.val == T::EMPTY
     }
 
+    /// Returns the flag that is set if there is exactly one.
+    #[inline(always)]
+    pub fn to_flag(self) -> Option<T> {
+        if _internal::BitFlagNum::count_ones(self.val) == 1 {
```

Comment:
Not sure why you're using a fully-qualified path here.

```suggestion
-        if _internal::BitFlagNum::count_ones(self.val) == 1 {
+       if self.val.is_power_of_two() {
```

Reply:
I was using the fully-qualified path because otherwise a use statement for `_internal::BitFlagNum` would have to be added. I was assuming it to be undesirable since it wasn't there.

Reply:
It wasn't there because there wasn't any use for that — the trait wasn't referenced outside of `_internal` at all.